vm/src/pyobject.rs

```diff
@@ -107,6 +107,7 @@ pub struct PyContext {
     pub exceptions: exceptions::ExceptionZoo,
     pub int_cache_pool: Vec<PyObjectRef>,
     tp_new_wrapper: PyObjectRef,
+    pub(crate) original_stdio: Option<(PyObjectRef, PyObjectRef, PyObjectRef)>,
```

Comment:
I think python expect to have `sys.__stdin__` and so on has original stdio objects. is this nessessary in spite of them?

Reply:
Not really, and this was all just in an effort to reduce the overhead of doing `sys.stdout.write("x")`, but I think it's actually fast enough that it doesn't really matter. I'll just do what CPython does and make file `sys.stdout` if the argument isn't present.